contrib/systemd/labgrid-coordinator.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Labgrid Coordinator
+After=network.target
+
+[Service]
+ExecStart=/usr/bin/labgrid-coordinator start --logformat=syslogd --cbdir /var/lib/labgrid-coordinator --config /etc/labgrid/coordinator.yaml
```

Comment:
There is no standard `labgrid-coordinator` wrapper.

We use:
```
[Unit]
Description=labgrid coordinator

[Service]
User=labgrid
WorkingDirectory=/srv/labgrid/coordinator/workdir
Environment="PYTHONUNBUFFERED=1"
ExecStart=/usr/ptx-venvs/labgrid-staging/bin/crossbar start --loglevel=debug --logformat=syslogd
Restart=always

[Install]
WantedBy=multi-user.target
```

Using the stop action is better, though.

Reply:
The PYTHONUNBUFFERED is useful to have immediate output in the journal.

Reply:
> There is no standard labgrid-coordinator wrapper.

Right, this only existed in my Debian package. I have added one now in `setup.py`.

> The PYTHONUNBUFFERED is useful to have immediate output in the journal.

Ok, however I would leave enabling this to the user (via a override file, see above) in case one needs it for debugging.

Reply:
This line should have an example on howto use crossbar from a separate virtualenv.